### PR TITLE
Catalogue item detail view

### PR DESCRIPTION
**Pull request by Macroz**
Detail view for catalogue items. See #866 

First merge #894

### CODE REVIEW

File: src/cljs/rems/administration/workflow.cljs [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                  [inline-info-field (str (text :t.administration/title)
+                                          " "
+                                          (str/upper-case (:langcode localization))) (:title localization)])
+                [[inline-info-field (text :t.administration/type) (:type license)]
+                 (when (:textcontent license)
+                   (case (:type license)
+                     "link" [inline-info-field (text :t.create-license/external-link) [:a {:target :_blank :href (:textcontent license)} (:textcontent license) " " [external-link]]]
+                     "text" [inline-info-field (text :t.create-license/license-text) (:textcontent license)]
+                     nil))]
+                (when (= "link" (:type license))
```
Comment (cscwooller) [inline src/cljs/rems/administration/workflow.cljs]: I don't get it why there is twice check if license type is link?

File: src/cljs/rems/administration/workflow.cljs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        :open? (<= (count rounds 5))
+        :collapse (into [:div]
+                        (for [round rounds]
+                          [round-view round]))}])))
+
+(defn licenses-view [licenses language]
+  [collapsible/component
+   {:id "licenses"
+    :title (text :t.administration/licenses)
+    :top-less-button? (> (count licenses) 5)
```
Comment (cscwooller) [inline src/cljs/rems/administration/workflow.cljs]: The limit of 5 is now hard coded in quite many places, should it be a def somewhere?

File: src/cljs/rems/administration/workflow.cljs (hunk 1 above)
Comment (Macroz) [reply src/cljs/rems/administration/workflow.cljs]: I thought it's more readable this day when there is one check per field.  I.e. there is a field for the type (that must do the typecase) and there is another field that is only shown for link licenses. Therefore two checks.

File: src/cljs/rems/administration/workflow.cljs (hunk 2 above)
Comment (Macroz) [reply src/cljs/rems/administration/workflow.cljs]: I don't think it's useful to put it anywhere yet.

There might be a case for a reusable collapse with this kind of functionality later. The difficulty is in that the regular collapse takes any component as child and does not know of any counts. Also the counts might actually be use case specific.

**Pull request merged**